core/pom.xml

```diff
@@ -17,7 +17,7 @@
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
         <java.target>1.8</java.target>
 
-        <jgit.version>5.2.2.201904231744-r</jgit.version>
+        <jgit.version>5.7.0.202003110725-r</jgit.version>
```

Comment:
Mhh I can remember that this might have included some incompatibilities, but i can't recall any specifics...furthermore nothing from the release notes jumps into my eyes:
- https://projects.eclipse.org/projects/technology.jgit/reviews/5.3.0-release-review
- https://projects.eclipse.org/projects/technology.jgit/reviews/5.4.0-release-review
- https://projects.eclipse.org/projects/technology.jgit/reviews/5.5.0-release-review
- https://projects.eclipse.org/projects/technology.jgit/reviews/5.6.0-release-review

Besides the deprecation warnings when compiling the tests it seems fine.